tensorflow/lite/micro/kernels/depthwise_conv_test.cc

```diff
@@ -27,12 +27,10 @@ static const int kMaxFilterChannels = 64;
 static const int kMaxBiasChannels = 64;
 
 template <typename T>
-TfLiteStatus ValidateDepthwiseConvGoldens(TfLiteTensor* tensors,
-                                          int tensors_size,
-                                          const T* expected_output_data,
-                                          T* output_data, int output_length,
-                                          TfLiteFusedActivation activation,
-                                          float tolerance = 1e-5) {
+TfLiteStatus ValidateDepthwiseConvGoldens(
+    TfLiteTensor* tensors, int tensors_size, const T* expected_output_data,
+    T* output_data, int output_length, TfLiteFusedActivation activation,
+    float tolerance = 1e-5, int dilation_width = 1, int dilation_height = 1) {
```

Comment:
I'd would be nice to re-organize this test similarly to conv_test to take in a DepthwiseConvParams instead of passing dilation width and height directly.  In the future if we want to test multiple strides we can do so easily, and it keeps our tests consistent with one another.